### PR TITLE
[FLINK-4286] Have Kafka examples that use the Kafka 0.9 connector

### PR DESCRIPTION
**Pull request by mylog00**
There were added examples of using connector for Kafka v0.9. Examples using 0.8 connector renamed to "ReadFromKafka08" and "WriteIntoKafka08" respectively. The pom.xml was modified: added config for separated packages Kafka08.jar and Kafka09.jar using "maven-jar-plugin" instead "maven-shade-plugin".

### CODE REVIEW

**Comment by zentol** [comment]
It would be neat if the example were more similar. Looking at the diff between the 0.8 and 0.9 examples there should only be 4-5 different lines (basically replace 08 by 09), but there are quite a few more due to formatting and such.

**Comment by mylog00** [comment]
I think it is independent examples and make it so similar not necessary. But if it important I can fix it and decrease difference. Thank you for review.

**Comment by tzulitai** [comment]
Thank you for the contribution @mylog00!

It isn't necessary, but I agree with @zentol that it would be nice to have them consistent with each other :-)

**Comment by mylog00** [comment]
I did examples more similar. Now difference only Kafka connector version and number of properties to configure consumer.

**Comment by mylog00** [comment]
https://travis-ci.org/mylog00/flink/builds/198855164
Link to my TravisCI build of branch [FLINK-4286]. With all passed tests.

**Comment by rmetzger** [comment]
Hi @mylog00, thanks a lot for your contribution.
I wonder if it makes more sense to just upgrade the existing Kafka examples to use Kafka 0.9 instead of having two examples that are almost identical?

**Comment by mylog00** [comment]
Hi @rmetzger, i delete example for Kafka v0.8 and left only for Kafka v0.9.

**Comment by DmytroShkvyra** [comment]
Hi @rmetzger could you review this again?

File: flink-examples/flink-examples-streaming/pom.xml [hunk 1]
```diff
@@ -360,6 +360,26 @@ under the License.
...
+								<manifestEntries>
+									<program-class>org.apache.flink.streaming.examples.kafka.ReadFromKafka09</program-class>
+								</manifestEntries>
+							</archive>
+							<includes>
+								<include>org/apache/flink/streaming/examples/kafka/ReadFromKafka09.class</include>
+								<include>org/apache/flink/streaming/examples/kafka/WriteToKafka09.class</include>
+							</includes>
+						</configuration>
+					</execution>
```
Comment (rmetzger) [inline flink-examples/flink-examples-streaming/pom.xml]: I don't think you can use the maven jar plugin for creating the kafka09 example. This won't include the needed dependencies into the jar.

File: flink-examples/flink-examples-streaming/pom.xml [hunk 2]
```diff
@@ -482,54 +502,19 @@ under the License.
...
-										<include>org/I0Itec/**</include>
-										<include>jline/**</include>
-										<include>com/yammer/**</include>
-										<include>kafka/**</include>
-									</includes>
-								</filter>
-							</filters>
-						</configuration>
-					</execution>
-					<execution>
```
Comment (rmetzger) [inline flink-examples/flink-examples-streaming/pom.xml]: What's the reasoning for deleting this section?

File: flink-examples/flink-examples-streaming/pom.xml (hunk 1 above)
Comment (mylog00) [inline flink-examples/flink-examples-streaming/pom.xml]: Yes. My bad. I did not check what examples can be executed via command line. And I shouldn't have had to remove fat-jar packager from pom.xml file (see next comment). But this PR has many conflicts, so I close It and will create new one.

**Comment by mylog00** [comment]
Many merge conflicts.

**Comment by rmetzger** [comment]
Thank you!

**Comment by rmetzger** [comment]
This pull request actually updates the Kafka connector examples, so I think there's no need to do anything here: https://github.com/apache/flink/pull/3911/files

**Pull request closed without merging**